### ISSUE TITLE
DataExporter: does not work with summaryRow

### ISSUE THREAD

**Issue by davidsan1982** [primefaces/primefaces#7141]
I have a project which uses primefaces 8.0 and I'd like to know why dataExporter doesn't work with summaryRow and if there is any plan to fix this bug soon or if there is any way to do this without coding my own exporter. 

Thank you very much.

**Comment by Rapster** [comment]
Please follow issue template, it's hard to follow if you're posting a defect or requesting a feature request

**Comment by melloware** [comment]
I really doubt SummaryRow will be supported in the Exporter without Sponsoring via PF PRO or providing a PR.  I looked into it and its not so easy to implement the way SummaryRows are done.

**Comment by davidsan1982** [comment]
> I really doubt SummaryRow will be supported in the Exporter without Sponsoring via PF PRO or providing a PR. I looked into it and its not so easy to implement the way SummaryRows are done.

It's a pitty to read that. Thanks for your soon answer anyway.

**Comment by Rapster** [comment]
It's a pity to give up that easy :D

**Comment by davidsan1982** [comment]
> It's a pity to give up that easy :D
Hehehe. It doesn't matter. My project is abandomed already . Maybe i'll do another one in a different way.🤷🏻‍♂️
